src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/envelopes/EnvelopeEventProcessor.java

```diff
@@ -200,28 +196,11 @@ private void logMessageParsed(IMessage message, Envelope envelope) {
         );
     }
 
-    private void logFeignException(
-        IMessage message,
-        Envelope envelope,
-        FeignException exception
-    ) {
-        String baseMessage = String.format(
-            "Failed to process message with ID %s. CCD response: %s",
-            message.getMessageId(),
-            exception.contentUTF8());
-
-        logErrorMessage(envelope, exception, baseMessage);
-    }
-
     private void logMessageProcessingError(IMessage message, Envelope envelope, Exception exception) {
         String baseMessage = String.format("Failed to process message with ID %s.", message.getMessageId());
 
-        logErrorMessage(envelope, exception, baseMessage);
-    }
-
-    private void logErrorMessage(Envelope envelope, Exception exception, String baseMessage) {
         String fullMessage = envelope != null
-            ? baseMessage + String.format(", Envelope ID: %s, File name: %s", envelope.id, envelope.zipFileName)
+            ? baseMessage + String.format(" Envelope ID: %s, File name: %s", envelope.id, envelope.zipFileName)
```

Comment:
~That comma still gives sense to sentence ;)~

Perhaps we should remove `.` from `baseMessage` and add it here for `fullMessage` if needed ;)